lua/lspconfig/configs/oxlint.lua

```diff
@@ -15,6 +15,27 @@ return {
     },
     root_dir = util.root_pattern('.oxlintrc.json'),
     single_file_support = false,
+
+    commands = {
+      OxcFixAll = {
+        function()
+          local client = util.get_active_client_by_name(0, 'oxlint')
```

Comment:
use `vim.lsp.get_client({name=...})`, there is no need for `get_active_client_by_name`

@dundargoc this would be an easy one to deprecate

Reply:
I couldn't find documentation of `vim.lsp.get_client` perhaps you meant `vim.lsp.get_clients`?